internal/core/streams/stream_testdata_test.go

```diff
@@ -49,13 +50,17 @@ type mockStoreQueries struct {
 	fakeSegmentsError error
 	flags             []ldstoretypes.KeyedItemDescriptor
 	segments          []ldstoretypes.KeyedItemDescriptor
+	latency           time.Duration
```

Comment:
Added this for testing concurrent calls to Replay - to ensure we have both calls use the same flightgroup.